CommunicationLayer/CommDeviceControl/RadioCommDevice.cpp

```diff
@@ -0,0 +1,55 @@
+#include <QDebug>
+#include <QTimer>
+#include "RadioCommDevice.h"
+
+namespace
+{
+   const QString DEFAULT_PORT_NAME = "/dev/pts/7";
```

Comment:
What do you mean by this? Would the ifdef differ pre platform or what what would be the thing that is defined?